test.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/bash -e
+#!/bin/bash -eux
 
-TEST_IMAGE='ruby:3.0'
+echo "==> Starting test.sh"
 
 rm -f Gemfile.lock
 
+echo "==> Docker Run"
+
 docker run --rm \
-  -v "$PWD:/usr/src/app" \
-  -w /usr/src/app \
-  -e CONJUR_ENV=ci \
-  $TEST_IMAGE \
-  bash -c "gem update --system && bundle update && bundle exec rake spec"
+    --volume $PWD:/usr/src/app \
+    --workdir /usr/src/app cyberark/ubuntu-ruby-builder \
+    bash -c 'git config --global --add safe.directory /usr/src/app && apt update && apt search libyaml && apt install libyaml-dev && gem update --system && bundle install && gem install spec && bundle install && bundle update && bundle exec rspec' || :
```

Comment:
should this default to true if it fails?

Reply:
@neil-k-zero How so?  Shouldn't test.sh report failure if the commands fail?

Reply:
Removed || :